### ISSUE TITLE
Sphinx app has no attribute debug

### ISSUE THREAD

**Issue by oliverw1** [spinus/sphinxcontrib-images#4]
Just installed sphinxcontrib-images and added the extension in my conf.py.
Upon issuing `make html`, I am greeted with the following exception:

```
oliver@armstrong:~/Documents/orders$ make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.1.3
loading pickled environment... done

Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/sphinxcontrib/images.py", line 236, in configure_backend
    app.debug("Backend passed as name: {}".format(backend_name_or_callable))
AttributeError: 'Sphinx' object has no attribute 'debug'
The full traceback has been saved in /tmp/sphinx-err-FtpPIk.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <http://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [html] Error 1
```

The traceback show the version numbers I'm using:
```
oliver@armstrong:~/Documents/orders$ cat /tmp/sphinx-err-FtpPIk.log
# Sphinx version: 1.1.3
# Python version: 2.7.3
# Docutils version: 0.9.1 release
# Jinja2 version: 2.6
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.1.3-py2.7.egg/sphinx/cmdline.py", line 188, in main
    warningiserror, tags)
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.1.3-py2.7.egg/sphinx/application.py", line 134, in __init__
    self._init_builder(buildername)
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.1.3-py2.7.egg/sphinx/application.py", line 195, in _init_builder
    self.emit('builder-inited')
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.1.3-py2.7.egg/sphinx/application.py", line 314, in emit
    results.append(callback(self, *args))
  File "/usr/local/lib/python2.7/dist-packages/sphinxcontrib/images.py", line 236, in configure_backend
    app.debug("Backend passed as name: {}".format(backend_name_or_callable))
AttributeError: 'Sphinx' object has no attribute 'debug'
```

In any case, /usr/local/lib/python2.7/dist-packages/sphinxcontrib/images.py has `app.debug` in several places, so just commenting this line won't suffice. In what version did the `debug` method appear? And when did it disappear?

**Comment by spinus** [comment]
#14 was merged, now sphinx 1.1 should be supported. It will be released soon.

**Comment by spinus** [comment]
@oliverw1 0.5.0 with sphinx 1.1 support released. https://pypi.python.org/pypi/sphinxcontrib-images/0.5.0
